content/docs/operating/security.md

```diff
@@ -116,9 +121,38 @@ resulting from additional load and failed scrapes.
 
 ## Authentication, Authorization, and Encryption
 
-Prometheus and its components do not provide any server-side
-authentication, authorization or encryption. If you require this, it is
-recommended to use a reverse proxy.
+In the future, server-side TLS support will be rolled out to the different
+Prometheus projects. Those projects include Prometheus, Alertmanager,
+Pushgateway and the official exporters.
+
+TLS client-side authentication with certificates will also be supported.
```

Comment:
This seems misleading, we already support that for clients. "Authentication of clients by TLS client certs" would be clearer.

Reply:
fixed wording